src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -95,9 +95,9 @@ public function boot(): void
     {
         $_ENV['DOCTRINE_DEPRECATIONS'] = $_SERVER['DOCTRINE_DEPRECATIONS'] ??= 'trigger';
 
-        $handler = ErrorHandler::register(null, false);
-
-        $this->container->get('debug.error_handler_configurator')->configure($handler);
```

Comment:
I confirm I don't like this approach: now, logs are not going to be handled the same way during tests.